app/assets/stylesheets/abstracts/_variables.scss

```diff
@@ -0,0 +1,3 @@
+$bg-gray-table: #435D7D;
```

Comment:
I think it would be part of the previous commit

If possible squash the commits since they are part of the same history  😄

Reply:
Okay I'll search info about squash commits, thanks